.yarnrc

```diff
@@ -0,0 +1 @@
+ network-timeout 600000
```

Comment:
Let's do it less, 600000 ms is very big and potential can freeze CI on long time

Reply:
what do you suggest? ;)

Reply:
https://github.com/yarnpkg/yarn/issues/5540#issuecomment-374069461

Reply:
hm, okay, sounds good